vault/data_source_pki_secret_backend_config_est_test.go

```diff
@@ -0,0 +1,49 @@
+package vault
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-provider-vault/internal/consts"
+	"github.com/hashicorp/terraform-provider-vault/internal/provider"
+	"github.com/hashicorp/terraform-provider-vault/testutil"
+)
+
+func TestAccDataSourcePKISecretConfigEst(t *testing.T) {
+	backend := acctest.RandomWithPrefix("tf-test-pki-backend")
+	dataName := "data.vault_pki_secret_backend_config_est.test"
+	resource.Test(t, resource.TestCase{
+		ProviderFactories: providerFactories,
+		PreCheck: func() {
+			testutil.TestAccPreCheck(t)
+			testutil.TestEntPreCheck(t)
+			SkipIfAPIVersionLT(t, testProvider.Meta(), provider.VaultVersion116)
+		},
+		Steps: []resource.TestStep{
+			{
+				Config: testPKISecretEmptyEstConfigDataSource(backend),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr(dataName, consts.FieldBackend, backend),
+					resource.TestCheckResourceAttrSet(dataName, consts.FieldEnabled),
```

Comment:
nit: can we validate that we get the expected defaults from Vault for these fields?
```suggestion
					resource.TestCheckResourceAttr(dataName, consts.FieldEnabled, "false"),
```

Reply:
This and the similar comment about AuditFields below, do we want the TFVP tests to start failing if Vault changes the default values? Seems a little brittle which is why these are a minimal set based checks but the full suite is validated within the resource check when the test actually sets all the values. 

If you feel strongly I have no issue adding them in, or maybe add a comment in this saying this is more thoroughly tested within the `TestAccPKISecretBackendConfigEst_basic` test?

Reply:
Definitely a nit, don't feel strongly about it! I'm happy with the comment you have added